package.json

```diff
@@ -96,23 +96,14 @@
     "rxjs": "^6.6.7",
     "zustand": "^4.1.1"
   },
-  "devDependencies": {
-    "@folio/stripes-cli": "3.2.2",
-    "@types/lodash": "^4.14.197",
-    "eslint": "^6.2.1",
-    "lodash": "^4.17.5"
-  },
+
   "resolutions": {
-    "@jridgewell/gen-mapping": "0.3.7",
-    "@types/react": "^18",
```

Comment:
We need to keep `@types/react` at `^18`. `react-intl` has a `*` dep on it and yarn kneecaps us, choosing to install _both_ the highest available version (`19.x`) to satisfy that `*` constraint, and also `18.x` to satisfy the various `^18` constraints specified in our own `package.json` files. Why it can't figure out that `18` would satisfy them both, I have no idea.